combat-trainer.lic

```diff
@@ -1895,6 +1926,9 @@ class GameState
     @fatigue_regen_threshold = settings.fatigue_regen_threshold
     echo("  @fatigue_regen_threshold: #{@fatigue_regen_threshold}") if $debug_mode_ct
 
+    @balance_regen_threshold = settings.balance_regen_threshold
```

Comment:
Can you add this to the declarations? Since it's 'read-only', it should probably be in the `attr_reader` list (which only generates a 'getter' function) on line 2420.